internal/services/redis/redis_linked_server_resource.go

```diff
@@ -176,6 +181,7 @@ func resourceRedisLinkedServerRead(d *pluginsdk.ResourceData, meta interface{})
 
 			d.Set("linked_redis_cache_location", location.Normalize(props.LinkedRedisCacheLocation))
 			d.Set("server_role", string(props.ServerRole))
+			d.Set("geo_replicated_primary_host_name", string(*props.GeoReplicatedPrimaryHostName))
```

Comment:
Right now this'll crash if a `props.GeoReplicatedPrimaryHostName` isn't returned (i.e. it's `null` in the response) - using `pointer.From` will default this to an empty string when it's unset - so can we update this:

```suggestion
			d.Set("geo_replicated_primary_host_name", pointer.From(props.GeoReplicatedPrimaryHostName))
```

Reply:
Done!